packages/react-magma-dom/src/components/CharacterCounter/CharacterCounter.test.js

```diff
@@ -126,23 +126,13 @@ describe('CharacterCounter', () => {
   });
 
   describe('accessibility', () => {
```

Comment:
I wonder if there's a way to add some unit tests for this fix, maybe by looking at the text inside `HiddenLabelText`?